server/pages/index.jade

```diff
@@ -152,18 +152,11 @@ html(lang=lang, dir=isRTL ? 'rtl' : 'ltr', class=isFluidWidth ? 'is-fluid-width'
 			script(type='text/javascript')!= config
 		if i18nLocaleScript
 			script(src=i18nLocaleScript)
-		if 'development' === env || isDebug
-			script(src=urls[ 'manifest' ])
-			script(src=urls[ 'vendor' ])
-			script(src=urls[ jsFile ])
-			if chunk
-				script(src=urls[ chunk ])
-		else
```

Comment:
without out this if / else how do we tell prod to serve minified files?

Reply:
nice catch! i need to revert this file now that we have separate `min` files again.